intl/hn/README.md

```diff
@@ -0,0 +1,43 @@
+# मैंने #100DaysOfCode चैलेंज में भाग लिया
+
+## विषय - सूची
+* [नियम](rules.md)
+* [लॉग - मेरी प्रगति देखने के लिए यहां क्लिक करें](log.md)
+* [साजिप्रा](FAQ.md)
+* [संसाधन](Resources.md)
+
+## अनुवाद
+[বাংলা](README.md) - [中文](../ch/README.md) - [deutsch](../de/README.md) - [español](../es/README.md) – [français](../fr/FAQ-fr.md) – [日本語](../ja/README.md) - [한국어](../ko/README-ko.md) – [norsk](../no/README.md) –  [polski](../pl/README.md) - [português do Brasil](../pt-br/LEIAME.md) - [русский](../ru/README-ru.md) – [українська](../ua/README-ua.md)
+
+यदि आप अपनी भाषा में अनुवाद सामग्री और नियमों की मदद करने में रुचि रखते हैं, तो 'इंटल' फ़ोल्डर के अंदर एक उप-फ़ोल्डर बनाएं और अपनी अनुवादित फ़ाइल को वहां रखें। फिर एक पुल अनुरोध करें (या ट्विटर @ DM11away पर DM)
+
+
+## यदि आप भाग लेना चाहते हैं:
+
+1. #100DaysOfCode की [आधिकारिक वेबसाइट] (http://100daysofcode.com/) पर जाएं। इस सूची में से अपनी पसंद के किसी भी मंच पर दूसरों से जुड़ें: www.100DaysOfCode.com/connect
+  इसके अलावा, [it] (https://join.slack.com/t/100xcode/sared_invite/zt-eivg7x1x-wgNPD77_u4GcUwZNT8Zgg) 100DaysOfCode के स्लैक चैनल के लिए लिंक लिंक।
```

Comment:
This link seems to be off. I think the latest one is: https://join.slack.com/t/100xcode/shared_invite/zt-eivg7x1x-wgNPDh7ug_u4GcUwZNT8Zg

Reply:
Changed it